eng/Versions.props

```diff
@@ -86,12 +197,6 @@
     <MicrosoftTestPlatformCLIPackageVersion>17.10.0-preview-24073-02</MicrosoftTestPlatformCLIPackageVersion>
     <MicrosoftTestPlatformBuildPackageVersion>17.10.0-preview-24073-02</MicrosoftTestPlatformBuildPackageVersion>
   </PropertyGroup>
-  <PropertyGroup>
-    <!-- Dependencies from https://github.com/dotnet/runtime -->
-    <SystemCodeDomPackageVersion>9.0.0-preview.2.24076.4</SystemCodeDomPackageVersion>
-    <SystemTextEncodingCodePagesPackageVersion>9.0.0-preview.2.24076.4</SystemTextEncodingCodePagesPackageVersion>
-    <SystemResourcesExtensionsPackageVersion>9.0.0-preview.2.24076.4</SystemResourcesExtensionsPackageVersion>
-  </PropertyGroup>
```

Comment:
These weren't removed, but moved up to the other `runtime` section that already exists. I tried to keep versions from the same repo in the same property group.